prisma/schema.prisma

```diff
@@ -116,6 +118,18 @@ model LocationWanted {
   profile Profile[]   @relation(references: [id])
 }
 
+model CompanySizes {
+  id                  Int               @id @default(autoincrement())
+  company             CompanySize  
+  profile Profile[]   @relation(references: [id])
```

Comment:
I'm not familiar with Prisma, so I'm trying to understand the syntax here.... It looks like we're using implicit many-to-many relations, in which case do we need `@relation(references: [id])`? In the example in the linked doc, it looks like they don't use it.

https://www.prisma.io/docs/concepts/components/prisma-schema/relations/many-to-many-relations#implicit-many-to-many-relations

Reply:
Also wondering why this is called `profile` and not `profiles` if it's a many-to-many relationship.